### DISCUSSION TITLE
收藏夹 列表 随机播放

### DISCUSSION THREAD

**Discussion by n0000000000o** [the1812/Bilibili-Evolved#4822] [功能建议 / Ideas]
场景：音乐收藏夹 500个视频，有时想听歌会“播放全部视频”，之后进入收藏夹页面，打开随机播放和自动切集，能明显感受到500个视频中 大概十几个视频会反复循环的播放，剩下的视频从来不会刷出来。疑似跟当前加载出的视频数量和随机播放的规则有关（甚至有时候顺序播放也会出现500个视频只会播放十几个的问题）。希望能有一个组件能解决相关问题。
感谢

**Comment by the1812** [comment]
这个应该找 b 站反馈下？
